.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
```

Comment:
can't you add this file to .gitignore or something? or not add it to your commit?